main.cpp

```diff
@@ -4303,6 +4308,7 @@ bool save_command::execute(device_map &devices) {
         fail(ERROR_NOT_POSSIBLE, "Save range crosses unmapped memory");
     }
     uint32_t size = end - start;
+    uint32_t chunk_size = std::max(size / 100, FLASH_SECTOR_ERASE_SIZE);
```

Comment:
I think i'd prefer chunk_size / 100 rounded up to the next FLASH_SECTOR_ERASE_SIZE multiple

Reply:
not critical for read, but perhaps tidier and more consistent with others